### PR TITLE
dnos6_command (New Module)

### PR DESCRIPTION
**Pull request by skg-net**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME
- modules/core/network/dnos6/dnos6_command.py
  ##### ANSIBLE VERSION
  ansible 2.2.0 (devel e8897a9b91) last updated 2016/09/09 20:20:56 (GMT +000)
  lib/ansible/modules/core: (detached HEAD bf5b3de83e) last updated 2016/09/09 20:21:38 (GMT +000)
  lib/ansible/modules/extras: (detached HEAD f83aa9fff3) last updated 2016/09/09 20:21:51 (GMT +000)
  config file =
  configured module search path = Default w/o overrides
  ##### SUMMARY
  Added Command module to support Dell Networking OS6

### CODE REVIEW

**Comment by skg-net** [comment]
@gundalow @privateip, please take a look at it. Thanks.

**Comment by gregdek** [comment]
Thanks @skg-net for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by skg-net** [comment]
Build is failing because, the dependent  PR https://github.com/ansible/ansible/pull/17474 is not merged.

File: network/dnos6/dnos6_command.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
+# !/usr/bin/python
```
Comment (privateip) [inline network/dnos6/dnos6_command.py]: remove extra space

Comment (skg-net) [reply network/dnos6/dnos6_command.py]: ready_for_review

**Pull request merged**